src/main/java/com/raisetech/worldCup2022/service/ParticipatingCountryServiceImpl.java

```diff
@@ -16,8 +18,9 @@ public ParticipatingCountryServiceImpl(ParticipatingCountryMapper participatingC
     }
 
     @Override
-    public List<ParticipatingCountry> findById(int id) {
-        return participatingCountryMapper.findById(id);
+    public ParticipatingCountry findById(int id) {
+        Optional<ParticipatingCountry> participatingCountry = participatingCountryMapper.findById(id);
+        return participatingCountry.orElseThrow(() -> new ResourceNotFoundException("resource not found:" + id));
```

Comment:
コロンの後ろは1つ半角スペースをいれるのが自然だと思います。
```suggestion
        return participatingCountry.orElseThrow(() -> new ResourceNotFoundException("resource not found: " + id));
```